cjwstate/modules/types.py

```diff
@@ -13,6 +13,14 @@
 from cjwstate.modules.param_dtype import ParamDType
 from .module_loader import validate_module_spec
 from .param_spec import ParamSpec
+from cjworkbench.i18n import supported_locales
```

Comment:
Do not import from `cjworkbench` into `cjwstate`. That's a circular dependency. (All our existing imports of this type are bugs.)